src/BloomExe/web/ServerBase.cs

```diff
@@ -17,6 +18,8 @@ namespace Bloom.web
 {
 	public abstract class ServerBase : IDisposable
 	{
+		internal const string SlashBloomSlash = "/bloom/";
```

Comment:
Two thoughts. One, why is this /bloom/ needed? Could you put a comment there? Two, this is an unusual variable name that just echos, etically, its value. Could it be something like "bloomPathPrefix"?